siriuspy/siriuspy/devices/lillrf.py

```diff
@@ -40,13 +42,31 @@ def amplitude(self, value):
 
     @property
     def phase(self):
-        """."""
+        """Phase."""
         return self['GET_PHASE']
 
     @phase.setter
     def phase(self, value):
         self['SET_PHASE'] = value
 
+    @property
+    def integral(self):
+        """Integral Enable."""
+        return self['GET_INTEGRAL_ENABLE']
+
+    @integral.setter
+    def integral(self, value):
+        self['SET_INTEGRAL_ENABLE'] = value
+
+    @property
+    def feedback(self):
+        """Feedback Mode."""
+        return self['GET_FB_MODE']
+
+    @feedback.setter
+    def feedback(self, value):
+        self['SET_FB_MODE'] = value
+
     def cmd_set_phase(self, value, timeout=10):
```

Comment:
I know this wasn't created by you, but I don't like this method. It is not a `cmd` maybe we coulde remove it and leave only the setter function. Maybe it would be useful to have commands to turn on and off the feedback and the integral. What do you think?

Reply:
do you think we can move the `wait` here to the property setter?

Reply:
> Maybe it would be useful to have commands to turn on and off the feedback and the integral. 

one command for each property? the command would wait for loop to be "well closed"?

Reply:
> do you think we can move the `wait` here to the property setter?

humm, I don't remember how we do it on other classes... maybe we could copy.

> one command for each property? 

yes

>the command would wait for loop to be "well closed"?

no, I don't think it is desirable for it to wait. But it would be useful to have a method to wait for it and also to check if the loop is well closed within some chosen range.

Reply:
@fernandohds564 can you please take a look at my first version for these methods?